package.json

```diff
@@ -16,77 +16,78 @@
   },
   "keywords": [],
   "dependencies": {
-    "ansi-diff": "0.0.0",
+    "ansi-diff": "^1.0.10",
     "ansi-escape-sequences": "^4.0.0",
     "async-collection": "^1.0.1",
     "babel-core": "^6.26.0",
     "babel-preset-env": "^1.6.1",
     "babelify": "^8.0.0",
-    "brfs": "^1.4.3",
+    "brfs": "^1.4.4",
     "brotli": "^1.3.2",
-    "browserify": "^15.0.0",
+    "browserify": "^15.2.0",
     "buffer-graph": "^4.0.0",
-    "clean-css": "^4.1.8",
+    "clean-css": "^4.1.9",
     "concat-stream": "^1.6.0",
     "css-extract": "^1.2.0",
-    "debug": "^3.0.1",
+    "debug": "^3.1.0",
     "dedent": "^0.7.0",
-    "disc": "^1.3.2",
-    "documentify": "^3.0.0",
+    "disc": "^1.3.3",
+    "documentify": "^3.1.0",
     "exorcist": "^1.0.0",
     "explain-error": "^1.0.4",
-    "fast-json-parse": "^1.0.2",
+    "fast-json-parse": "^1.0.3",
     "findup": "^0.1.5",
     "flush-write-stream": "^1.0.2",
-    "fs-compare": "0.0.4",
+    "fs-compare": "^0.0.4",
     "get-port": "^3.2.0",
     "glslify": "^6.1.0",
-    "gzip-size": "^3.0.0",
-    "hstream": "^1.0.0",
+    "gzip-size": "^4.1.0",
+    "hstream": "^1.1.0",
     "html-minifier": "^3.5.3",
+    "html-minifier": "^3.5.8",
```

Comment:
oops, the html-minifier line got duplicated somehow 🙈

Reply:
You are right !!!